schema/index.js

```diff
@@ -101,7 +102,8 @@ const schema = new GraphQLSchema({
     fields: {
       followArtist: FollowArtist,
       updateCollectorProfile: UpdateCollectorProfile,
-      updateConversation: UpdateConversation,
+      updateBuyerOutcome: UpdateBuyerOutcome,
+      appendConversationThread: AppendConversationThread,
```

Comment:
I guess we could go ahead and make an `operation` for the existing two conversation-related mutations but it might be overkill at the moment?